packages/template-tag-codemod/src/index.ts

```diff
@@ -97,22 +97,21 @@ export async function ensurePrebuild() {
 }
 
 export async function ensureAppSetup() {
-  let pkg: Package;
+  let filename = resolve(process.cwd(), 'package.json');
+  let content: string;
   try {
-    pkg = resolverLoader.resolver.packageCache.get(process.cwd());
+    content = readFileSync(filename, 'utf8');
```

Comment:
this no longer serves the purpose of ensuring an ember app
I had fun digging into what it was actually doing because it was buried very deep inside resolver code but it actually ensures ember-source is in package json which may not be the case for a v2 addon but I wonder if that was the intention here

Reply:
That sounds like a good case to add to the test suite. I'm assuming we want to encourage people to be able to use this to update a v2 addon?